runtime/v2/runc/service.go

```diff
@@ -382,14 +386,14 @@ func (s *service) Exec(ctx context.Context, r *taskAPI.ExecProcessRequest) (*pty
 	if err != nil {
 		return nil, errdefs.ToGRPC(err)
 	}
+	s.mu.Lock()
```

Comment:
Should this check existence again after the lock is reacquired or is that condition not possible?